library/net.c

```diff
@@ -737,7 +736,13 @@ int dnet_process_recv(struct dnet_net_state *st, struct dnet_io_req *r) {
 
 		dnet_access_context_add_string(context, "access", "server");
 		HANDY_COUNTER_INCREMENT("io.cmds", 1);
-		err = dnet_process_cmd_raw(st, cmd, r->data, 0, r->queue_time, context);
+
+		if (r->n2_c) {
```

Comment:
Use ternary operator.

Reply:
Ternary operator is good for short and simple conditions. Here can be problem with style. At my previous workplace we used style:
```
cond
    ? true_branch
    : false_branch;
```
But elliptics has another style, and I see only short conditions in the project.

Reply:
https://github.com/interiorem/elliptics/blob/e5c8cbece648a33b8650a0f3adcb0ba9ddb3bd92/bindings/cpp/result_entry.cpp#L134-L136